### PR TITLE
[Add-ins] (delegate) Note sync behavior

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[e890cff](https://github.com/OfficeDev/outlook-dev-docs/pull/655/commits/e890cff99bbe54e2f2033d9add20251c66f83944)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/delegate-access.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-delegate/docs/add-ins/delegate-access.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C29%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201907292218475199-655%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=oJwxGYKpbtIcjqXdgSUiw8CzhJjl09g%2FI6KKIGw1Zoc%3D&st=2019-07-29T22%3A14%3A44Z&se=2019-08-29T22%3A19%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[8ee2646](https://github.com/OfficeDev/outlook-dev-docs/pull/655/commits/8ee264646a7ac3950518f0eaefcd8b8475ef893a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/delegate-access.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-delegate/docs/add-ins/delegate-access.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C30%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201907301739171741-655%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gdOkLH4x6bE5VoZJH5q58b5nApz5f%2BTb0HEM%2BqTuLrE%3D&st=2019-07-30T17%3A35%3A14Z&se=2019-08-30T17%3A40%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/delegate-access.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ localization_priority: Normal
 A mailbox owner can use the delegate access feature to [allow someone else to manage their mail and calendar](https://support.office.com/article/allow-someone-else-to-manage-your-mail-and-calendar-41c40c04-3bd1-4d22-963a-28eafec25926). This article specifies which delegate permissions the Office JavaScript API supports and describes how to enable delegate access scenarios in your Outlook add-in.
 
 > [!IMPORTANT]
-> Delegate access for Outlook add-ins is currently [in preview](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview) and only supported in clients on Office build 16.0.10712.10000 or above and with their mailbox on Exchange Online. Delegate access APIs should not yet be used in production environments.
+> Delegate access for Outlook add-ins is currently [in preview](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview) and only supported in clients on Office build 16.0.10712.10000 or above and with their mailbox on Exchange Online. Delegate access APIs shouldn't be used in production environments yet.
```
Comment (ElizabethSamuel-MSFT) [inline docs/add-ins/delegate-access.md]: Is the build info true for Mac and modern OWA as well?

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[00cc8ca](https://github.com/OfficeDev/outlook-dev-docs/commits/00cc8caf5e2e0b84b0b2cede21cac73a70c6675d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/delegate-access.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-delegate/docs/add-ins/delegate-access.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C5%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201908052104519174-655%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=vBQYxotzqaA2MQRBFOf2v%2BhXLXg%2Fb8lXz2OVwHcvS0A%3D&st=2019-08-05T21%3A00%3A52Z&se=2019-09-05T21%3A05%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**